src/Vault.sol

```diff
@@ -378,17 +384,22 @@ contract Vault {
      */
     function liquidateVault(address liquidationKeeper) public onlyFactory returns (bool success, address liquidator_) {
```

Comment:
Function is now very inefficient implemented, both for totalValue as for liquidityThreshold we now loop over all assets and retrieve the value for each asset.
Also the liquidation threshold should be defined at the moment a position is modified, it should not dynamically vary with the composition of the vault. Users should have a fixed liquidation value.